x-pack/heartbeat/scenarios/framework/framework.go

```diff
@@ -40,24 +40,32 @@ type Scenario struct {
 	NumberOfRuns int
 }
 
-type Twist func(Scenario) Scenario
+type Twist struct {
+	Name string
+	Fn   func(Scenario) Scenario
+}
 
-func MakeTwist(name string, fn Twist) Twist {
-	return func(s Scenario) Scenario {
-		newS := s.clone()
-		newS.Name = fmt.Sprintf("%s~<%s>", s.Name, name)
-		return fn(newS)
+func MakeTwist(name string, fn func(Scenario) Scenario) *Twist {
+	return &Twist{
+		Name: name,
+		Fn: func(s Scenario) Scenario {
+			newS := s.clone()
+			newS.Name = fmt.Sprintf("%s~<%s>", s.Name, name)
+			return fn(newS)
+		},
 	}
 }
 
-func MultiTwist(twists ...Twist) Twist {
-	return func(s Scenario) Scenario {
-		res := s
-		for _, twist := range twists {
-			res = twist(res)
-		}
-		return res
-	}
+func MultiTwist(twists ...*Twist) *Twist {
+	return MakeTwist(
+		"<~MULTI-TWIST~[",
```

Comment:
Makes things more legible in the console

Reply:
I'd suggest moving this comment into the code for future developers' convenience.

 ```suggestion
		"<~MULTI-TWIST~[", // Makes things more legible in the console
```